libs/storage/Tsavorite/cs/src/core/Index/Tsavorite/Implementation/Revivification/FreeRecordPool.cs

```diff
@@ -575,7 +575,7 @@ internal void ScanForEmpty(CancellationToken cancellationToken)
         public void Dispose()
         {
             foreach (var bin in bins)
-                bin.Dispose();
+                FreeRecordBin.Dispose();
```

Comment:
This loop becomes a no-op with this change. Better to remove FreeRecordBin.Dispose entirely as there's currently no need for it